packages/openactive-broker-microservice/app.js

```diff
@@ -813,7 +813,7 @@ function processOpportunityItem(item) {
 /** @type {RpdePageProcessor} */
 function monitorOrdersPage(rpde) {
   rpde.items.forEach((item) => {
-    if (item.data && item.id && orderResponses[item.id]) {
+    if (item.id && orderResponses[item.id]) {
```

Comment:
Required to allow deletes to be picked up by the `/get-order/:orderUuid` endpoint